docs/WritingFirstTest-README.md

```diff
@@ -0,0 +1,97 @@
+# Writing the first tests
+
+## Pre-requisites:
+
+        1. Have admin rights  
+        2. Intellij (install cucumber JVM plugin)
+        3. Java JDK 11  
+        4. Gradle 7.1
+        5. Appium
+        6. Android studio with all paths set
+        7. GIT
+        8. Docker 
+        9. Create a fork and Clone the repository from - https://github.com/znsio/teswiz.
+
+## Flow understanding
+
+    # Feature Files:    
+            These are the files that contains the tests in BDD format with the tags related to 
+            platform on which the tests will be running. Tags such as @android or @web etc. 
+            Also, there can be more tags related to features can be added in feature level.
+    
+    # Steps File:
+            These are files that have the definition of the steps that is used in feature files. 
+            LOGGER can be added in each method. From steps file, there would be a call made to BL.
+    
+    # Business Layer:
+            These files have the logic to verify the assertions or business logics to verify. 
+            From the BL, there will be call made to screens or the APIs. This will be the logical 
+            layer for the suite. For each screen there would be a separate BL. 
+    
+    # Screens: 
+            These are the files there will be the screen navigation and locators for each page will
+            be here. The methods shouldn’t have void return type. 
+
+    # Services: 
+            These are the files that contain the API service calls required for pre-requisite data.
+    
+    # Caps: 
+            This directory contains the capabilities required for android devices such as app 
+            activity and app package.
+
+    # Configs: 
+            This directory has the information about the configuration related to browser, parallel
```

Comment:
The config.properties file has configuration details related to the execution of the tests.